hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/FixedFileTrailer.java

```diff
@@ -121,7 +122,8 @@ public class FixedFileTrailer {
    * Raw key comparator class name in version 3
    */
   // We could write the actual class name from 2.0 onwards and handle BC
-  private String comparatorClassName = CellComparator.getInstance().getClass().getName();
+  private String comparatorClassName =
```

Comment:
No compatibility issues?

Reply:
> No compatibility issues?

Thought about it and i think so, because we do the conversion when we save and read the hfile, the actual store is the KVComparator, we can't change this since we want hbase1.x to be able to read the files generated in the new version.
Thanks.